prow/cmd/sinker/main.go

```diff
@@ -68,6 +69,16 @@ func main() {
 		logrus.WithError(err).Fatal("Error starting config agent.")
 	}
 
+	clusterConfig, err := rest.InClusterConfig()
```

Comment:
I think we should continue supporting the existing auth methods until everything has migrated.

We can just encapsulate this into something reusable:

https://github.com/kubernetes/test-infra/blob/aa8d6d1c02a9deb17f3c631a4ac1dc87885fbdb7/prow/cmd/build/main.go#L157-L194

Reply:
The PJ client was always in cluster, though. I don't understand you.